### DISCUSSION TITLE
{Feature: Upcoming Events} 【Main-Issue】Figma Desing Creation

### DISCUSSION THREAD

**Discussion by JeanSanchezFelix** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#318] [Polls]
**Tasks:**

- **Figma Design:**
  - Create an initial Figma design for the event creation pop-up component.
  - Share designs for review with the team.
  - Open a poll to vote on the preferred design.
  - *Reminder: Please refer to the existing components for guidance in your Figma design.*  
    [Figma design link](https://www.figma.com/design/0N1lAA4QRe7vLjmEjBPkcG/Eventero?node-id=12-474)

- **Poll for Figma Design:**
  - Once Figma designs are shared, a poll will be created to allow the team to vote for the final design.

Feel free to contribute any thoughts or additional tasks in this thread. Let’s make this a smooth transition from the homepage to the event creation component!

_Originally posted by @AlexandraFuentes in https://github.com/uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management/discussions/240_

**Comment by LuisJCruz** [comment]
Here is the figma design I created for the event creation pop-up. Let me know your thoughts! 
[https://www.figma.com/design/W9vD0FjNh8SQMZTi2tTlA6/Eventero-2?node-id=0-1&t=iLi6GA1HkpyY2gj0-1](url)

**Reply by AlexandraFuentes** [reply]
Wow! love the design
